.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: Install Protoc
-        uses: arduino/setup-protoc@v1
+        uses: arduino/setup-protoc@v3
```

Comment:
Is this change expected?

Reply:
yeah, the link in the old version no longer exists. I bump it's version to let the ci pass.